lib/HttpClient/CurlClient.php

```diff
@@ -117,39 +124,39 @@ public function request($method, $absUrl, $headers, $params, $hasFile)
         if (is_callable($this->defaultOptions)) { // call defaultOptions callback, set options to return value
             $opts = call_user_func_array($this->defaultOptions, func_get_args());
             if (!is_array($opts)) {
-                throw new Error\Api("Non-array value returned by defaultOptions CurlClient callback");
+                throw new \Stripe\Error\Api("Non-array value returned by defaultOptions CurlClient callback");
```

Comment:
I like that a lot of these symbols are now fully qualified! Much easier to follow now.